css/style.css

```diff
@@ -37,15 +35,17 @@ label {
     margin-left: auto;
     margin-right: auto;
     margin-bottom: 1.5em;
+    margin-top: 0.2em;
 }
 
 #submit {
```

Comment:
Add hover effects to buttons
![image](https://user-images.githubusercontent.com/67667688/192340561-0eda249c-8d6b-4770-84c0-874a3b87a05d.png)